data/MadhoolikaNagesh.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Madhooolika Nagesh",
+  "displayStatsPublic":true,
```

Comment:
hey @MadhoolikaNagesh currently you don't need to add `"displayStatsPublic":true,`. You can delete this one.
```suggestion

```